namer/serversets/src/test/scala/io/l5d/ServersetNamerTest.scala

```diff
@@ -42,7 +42,7 @@ class ServersetNamerTest extends FunSuite {
   }
 
   def namer(path: String) = new ServersetNamer("host") {
-    val loopback = new InetSocketAddress(InetAddress.getLoopbackAddress, 1)
+    val loopback = Address(new InetSocketAddress(InetAddress.getLoopbackAddress, 1))
```

Comment:
Hmm, any idea why this compilation failure didn't trigger when we upgraded finagle?  This looks like the right fix, but just curious.